suite-native/atoms/src/Chip.tsx

```diff
@@ -65,6 +68,7 @@ export const Chip = ({
     onSelect,
     description,
     icon,
+    titleColor,
```

Comment:
gray800 should be here as default value instead of style

Reply:
[41b0034](https://github.com/trezor/trezor-suite/pull/6549/commits/41b003498353846fafe757895944d128ebe7dc33)